packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <span
+    data-qa="dt-split-button"
+    class="d-split-btn"
+    :style="{ width: width }"
+  >
+    <dt-button
+      v-dt-tooltip="alphaTooltipText"
+      data-qa="dt-split-button-alpha"
+      :size="size"
+      :class="[`d-split-btn__alpha d-split-btn__alpha--${size}`]"
+      :active="alphaActive"
+      :aria-label="alphaAriaLabel"
+      :icon-position="alphaIconPosition"
```

Comment:
Bit of an edge case, but if icon-position is set and there is no icon, it still re-arranges the button.

![2024-07-30 12 27 57](https://github.com/user-attachments/assets/a41cd5d4-ff42-4df1-b1d4-bb017f0afa29)

Reply:
That needs to be fixed on the actual DtButton component, split-button its just passing through that prop. I'll create a ticket.

Reply:
https://dialpad.atlassian.net/browse/DLT-1931

Reply:
Thanks